media-libs/libwebm/libwebm-1.0.0.27.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="WebM VP8 Coced SDK"
+DESCRIPTION="libwebm - WebM VP8 Codec SDK"
+HOMEPAGE="https://www.webmproject.org/"
+COPYRIGHT="2010 Google Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/webmproject/libwebm/archive/libwebm-$portVersion.tar.gz"
+CHECKSUM_SHA256="1332f43742aeae215fd8df1be6e363e753b17abb37447190e789299fe3edec77"
+SOURCE_DIR="libwebm-libwebm-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libwebm$secondaryArchSuffix = $portVersion
+	cmd:dumpvtt$secondaryArchSuffix
+	cmd:vttdemux$secondaryArchSuffix
+	lib:libwebm$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libwebm${secondaryArchSuffix}_devel = $portVersion
+	devel:libwebm$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	libwebm$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage libwebm$secondaryArchSuffix \
+	"$binDir"/dumpvtt \
+	"$binDir"/vttdemux \
+	"$libDir"/libwebm.so
+
+BUILD()
+{
+	make all -f Makefile.unix $jobArgs
+	make -f Makefile.unix shared $jobArgs
```

Comment:
You might wish to combine the two calls:
```diff
-	make all -f Makefile.unix $jobArgs
-	make -f Makefile.unix shared $jobArgs
+	make -f Makefile.unix all shared $jobArgs
```
(This is not always equivalent, but here we can do so. I've compared two builds (i.e. before and after the change) to make sure it was OK.)